engines/director/lingo/lingo-the.cpp

```diff
@@ -1105,7 +1105,9 @@ void Lingo::setTheEntity(int entity, Datum &id, int field, Datum &d) {
 		break;
 	case kTheTimeoutLapsed:
 		// timeOutLapsed can be set in D4, but can't in D3. see D3.1 interactivity manual p312 and D4 dictionary p296.
-		setTheEntitySTUB(kTheTimeoutLapsed);
+		if (g_director->getVersion() >= 400 && d.type == INT) {
```

Comment:
What happens when it is not  `INT`, shall we probably print out a warning? How does the original behave?